WebDriverAgentLib/Categories/XCUIElement+FBTap.m

```diff
@@ -36,6 +41,11 @@ - (BOOL)fb_tapWithError:(NSError **)error
 
 - (BOOL)fb_tapCoordinate:(CGPoint)relativeCoordinate error:(NSError **)error
 {
+  if (SYSTEM_VERSION_GREATER_THAN_OR_EQUAL_TO(@"11.0")) {
+    XCUICoordinate *tapCoordinate = [[XCUICoordinate alloc] initWithElement:self normalizedOffset:CGVectorMake(relativeCoordinate.x, relativeCoordinate.y)];
+    [tapCoordinate tap];
```

Comment:
Ok, I'll change that. My point is that usage of public API is preferred over the private one (and events synthesiser used here is the private API) and we will be happy to get rid of the workarounds as soon as we stop supporting iOS9/10 at some point of time.